File: app/src/main/java/com/hjl/artisan/tool/view/EdModelViewHolder.java
package com.hjl.artisan.tool.view;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;
import com.hjl.artisan.R;

public class EdModelViewHolder extends RecyclerView.ViewHolder{
    public TextView tv;
    public EditText ed;
    public EdModelViewHolder(View itemView) {
        super(itemView);
        ed= (EditText) itemView.findViewById(R.id.edModelItem);
        tv= (TextView) itemView.findViewById(R.id.tvModelItem);
    }
}

File: app/src/main/java/com/hjl/artisan/tool/view/ActualMeasurement/EdModelViewHolder.kt
package com.hjl.artisan.tool.view.ActualMeasurement

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.EditText
import android.widget.TextView
import com.hjl.artisan.R

class EdModelViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var tv: TextView = itemView.findViewById<View>(R.id.tvModelItem) as TextView
    var ed: EditText = itemView.findViewById<View>(R.id.edModelItem) as EditText
}
